blacklight.gemspec

```diff
@@ -25,15 +25,15 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '~> 2.1'
 
-  s.add_dependency "rails", "~> 5.1"
+  s.add_dependency "rails", '>= 5.1', '< 7'
   s.add_dependency "globalid"
   s.add_dependency "jbuilder", '~> 2.7'
   s.add_dependency "kaminari", ">= 0.15" # the pagination (page 1,2,3, etc..) of our search results
   s.add_dependency "deprecation"
 
   s.add_development_dependency "rsolr", ">= 1.0.6", "< 3"  # Library for interacting with rSolr.
   s.add_development_dependency "solr_wrapper"
-  s.add_development_dependency "rspec-rails", "~> 3.5"
+  s.add_development_dependency "rspec-rails", "~> 4.0.0.beta2"
```

Comment:
Should we wait for the official 4.0.0 release?  Maybe it won't be too long from now (https://github.com/rspec/rspec-rails/pull/2071#issuecomment-532519693)?

Reply:
I'm not sure this is a big concern of mine since this is just a development dependency and it works for us.